### DISCUSSION TITLE
Support for Gnome 45?

### DISCUSSION THREAD

**Discussion by dimmollo** [material-shell/material-shell#979] [General]
I'm a big fan of Material Shell and it makes my workflow much easier and more efficient. Now Gnome 45 is coming, and from what I read, it changes quite a bit how extensions work. Is Material Shell going to be compatible with Gnome 45?

**Comment by PapyElGringo** [comment]
Hey there ! I was checking the changes and there indeed a lot of mandatory changes requested to be compatible to version 45...

Since I'm current focus is to work on our standalone Window Manager I'm don't know yet if I'll priorise this over progress on the WM!

**Reply by dimmollo** [reply]
Hey, thanks for the reply.

That makes sense, I will just stay on Gnome 44 until the WM comes along then. All the best for the project!

**Comment by xalgia** [comment]
First of all @PapyElGringo Much appreciation for creating the most incredible and performance-rich desktop experience -- thank you for creating material shell. I consider your creation legendary compared to every other DE and WM out there. 

May we know how long until the standalone WM will be ready for beta daily driving?

**Reply by PapyElGringo** [reply]
Hard to tell I guess It's will highly depend on the success of the Crowdfunding campaign meant to start on November

**Comment by AskAlice** [comment]
@PapyElGringo I support you creating a fully-fledged WM for this concept, and I don't want to impede your time delegation, 
so this goes out to anyone interested: for the sake of maintainence I'll put $100 bounty on this, for anyone to claim as long as there is a usable or semi-usable fork/branch with the intent of merging mainline

**Reply by PapyElGringo** [reply]
Thanks It's very generous of you !

**Comment by janhampe** [comment]
I know nothing about Typescript, so I just wanted to ask. How much work do you think it would be to migrate to GNOME 45. I have been daily driving MS for two years now, every since I got my laptop, and now that it broke (and I can't figure out how to downgrade to 44) things have been bad. So if anyone could do this, I'd be happy to help, but I don't think I have the skills to do it alone.

**Reply by AskAlice** [reply]
I took a look at it as I write typescript by day and well I think this requires some special knowledge with this gnome extension development, and with this codebase in particular to get working. I spent maybe a half hour or an hour messing around and I got blocked when I couldn't figure out how to get types from gnome's extension library to show up in this repo.

**Reply by PapyElGringo** [reply]
Please send me caffeine pills 😪

https://github.com/material-shell/material-shell/pull/984

**Reply by AskAlice** [reply]
~~sent via paypal, though it said you don't have an account.~~ ill put it in the crowdfund :+1:

**Reply by PapyElGringo** [reply]
🙏

**Comment by PapyElGringo** [comment]
In order to allow us to be able to keep the Material Shell Workflow until Veshell is ready I spend 3 days (and part of my nights) for making Material Shell compatible to gnome 45 https://github.com/material-shell/material-shell/pull/984

Can you guys test it out?

**Reply by PapyElGringo** [reply]
@dimmollo @SirZayers @AskAlice @xalgia

**Reply by dimmollo** [reply]
Oh, that's great! Unfortunately I don't have another suitable machine on which to test it, so if others can help it would be better. Otherwise I can try and see if I can do some testing on my raspberry pi over the weekend.

By the way, did the crowdfunding for Veshell start yet?

**Reply by PapyElGringo** [reply]
Yep it's available there https://opencollective.com/free-explorers/projects/veshell/contribute/kickstart-veshell-67471 🤗

**Reply by dimmollo** [reply]
Nice, I'll make sure to contribute very soon! :)

**Reply by EmileTrotignon** [reply]
Hey @PapyElGringo
I have tried to test your branch on manjaro with gnome 45, when I log in I get the material shell logo, but it disappears afterward. I guess it is crashing, and I would really like to give a log or error message, but I do not know how to see it.

**Reply by PapyElGringo** [reply]
`journalctl /usr/bin/gnome-shell -e`

**Reply by janhampe** [reply]
Thank you so much. I just installed it and everything seems to be working fine.
GNOME Shell 45.1
Arch Linux Kernel 6.5.9-arch2-1

**Comment by shawny43** [comment]
Said merged but it's still not published on Gnome extensions yet.
